css/common.css

```diff
@@ -13,8 +13,23 @@
     margin-bottom: 5px;
     z-index: 1;
 }
-
+.list-style-none{
+    list-style: none;
+}
 
 .gap-sm{
     gap:4px;
+}
+.card.p-2{
+    padding: 2rem;
+}
+
+.br-1px{
+    border: 1px solid;
+}
+
+.cart-count-num{
+    background: #efefef;
```

Comment:
Prefer to use variables

Reply:
thanks for pointing out 👍
i will fix that